lib/beacon/live_admin/components/admin_components.ex

```diff
@@ -87,6 +87,9 @@ defmodule Beacon.LiveAdmin.AdminComponents do
     <li class="mr-2">
       <.link patch={beacon_live_admin_path(@socket, @site, "/pages/#{@page_id}/revisions")} class={if(@current_action == :revisions, do: @active_class, else: @regular_class)}>Revisions</.link>
     </li>
+    <li class="mr-2">
+      <.link navigate={beacon_live_admin_path(@socket, @site, "/pages/#{@page_id}/variants")} class={if(@current_action == :variants, do: @active_class, else: @regular_class)}>Variants</.link>
+    </li>
```

Comment:
I think you can move these lines before "Revisions" so Layouts keeps the same pattern leaving "Revisions" at the end.